src/requirementslib/models/project.py

```diff
@@ -70,155 +66,3 @@ def dumps(self):
         strio = io.StringIO()
         self.model.dump(strio)
         return strio.getvalue()
-
```

Comment:
The Lockfile class has references to Project files which should be cleaned too.

Reply:
@oz123 `ProjectFile` is still a thing as its used in some capacity, its `Project` that I removed -- I don't see reference to `Project` in lockfile.py ...

Reply:
Ok. Sorry. I misread it.